benches/read_many_structs.rs

```diff
@@ -438,7 +438,7 @@ mod benchmark {
             let mut context = EncodingContext::for_ion_version(IonVersion::v1_1);
             context
                 .macro_table_mut()
-                .add_macro(compiled_macro.clone())
+                .add_template_macro(compiled_macro.clone())
```

Comment:
🪧 This method (which is specific to template macros) was renamed to accommodate adding a more general purpose `add_macro` method alongside it.